locale/ForemanDocker.pot

```diff
@@ -0,0 +1,633 @@
+msgid "Attach STDIN"
+msgstr ""
+
+#: ../foreman-docker/app/views/containers/steps/environment.html.erb:8
+msgid "Attach STDOUT"
+msgstr ""
+
+#: ../foreman-docker/app/views/containers/steps/environment.html.erb:9
+msgid "Attach STDERR"
+msgstr ""
+
+#: ../foreman-docker/app/views/containers/steps/environment.html.erb:12
+msgid "Environment variables"
+msgstr ""
+
+#: ../foreman-docker/app/views/containers/steps/environment.html.erb:16
+msgid "Add environment variable"
+msgstr ""
+
+#: ../foreman-docker/app/views/containers/steps/environment.html.erb:19
+msgid "Exposed Ports"
+msgstr ""
+
+#: ../foreman-docker/app/views/containers/steps/environment.html.erb:23
+msgid "Add Exposed Port"
+msgstr ""
+
+#: ../foreman-docker/app/views/containers/steps/environment.html.erb:25
+#: ../foreman-docker/app/views/containers/show.html.erb:57
+msgid "DNS"
+msgstr ""
+
+#: ../foreman-docker/app/views/containers/steps/environment.html.erb:29
+msgid "Add DNS"
+msgstr ""
+
+#: ../foreman-docker/app/views/containers/steps/environment.html.erb:35
+msgid "Run ?"
+msgstr ""
+
+#: ../foreman-docker/app/views/containers/steps/environment.html.erb:38
+msgid "If selected, the container will start after it is created"
+msgstr ""
+
+#: ../foreman-docker/app/views/containers/steps/_title.html.erb:1
+#: ../foreman-docker/app/views/containers/index.html.erb:4
+#: ../foreman-docker/lib/foreman_docker/engine.rb:51
+msgid "New container"
+msgstr ""
+
+#:
+#: ../foreman-docker/app/views/containers/steps/_form_buttons.html.erb:4
+#: ../foreman-docker/app/views/containers/show.html.erb:176
+msgid "Cancel"
+msgstr ""
+
+#:
+#: ../foreman-docker/app/views/containers/steps/_form_buttons.html.erb:8
+msgid "Back"
+msgstr ""
+
+#:
+#: ../foreman-docker/app/views/containers/steps/_form_buttons.html.erb:14
+#: ../foreman-docker/app/views/containers/show.html.erb:178
+msgid "Submit"
+msgstr ""
+
+#:
+#: ../foreman-docker/app/views/containers/steps/_form_buttons.html.erb:16
+msgid "Next"
+msgstr ""
+
+#: ../foreman-docker/app/views/containers/steps/image.html.erb:9
+msgid "Content View"
+msgstr ""
+
+#: ../foreman-docker/app/views/containers/steps/image.html.erb:15
+msgid "Docker hub"
+msgstr ""
+
+#: ../foreman-docker/app/views/containers/steps/image.html.erb:19
+msgid "External registry"
+msgstr ""
+
+#: ../foreman-docker/app/views/containers/steps/preliminary.html.erb:6
+msgid "Compute resource"
+msgstr ""
+
+#: ../foreman-docker/app/views/containers/steps/preliminary.html.erb:19
+msgid "Resource selection"
+msgstr ""
+
+#: ../foreman-docker/app/views/containers/steps/preliminary.html.erb:20
+msgid "Where do you want to deploy your container?:"
+msgstr ""
+
+#: ../foreman-docker/app/views/containers/steps/preliminary.html.erb:22
+msgid "Deploy on"
+msgstr ""
+
+#: ../foreman-docker/app/views/containers/steps/preliminary.html.erb:25
+msgid "You need a Docker compute resource in order to create containers. Please %s and try again."
```

Comment:
It's a link with the text "add a new one", that links to `hash_for_new_compute_resource_path`.